src/RulesDeploy.js

```diff
@@ -51,21 +51,43 @@ RulesDeploy.prototype = {
   /**
    * Create rulesets for each file added to this deploy, and record
    * the name for use in the release process later.
+   *
+   * If the ruleset to create is identical to the latest existing ruleset,
+   * then we record the existing ruleset name instead of creating a duplicate.
    */
-  createRulesets: function() {
+  createRulesets: function(service) {
```

Comment:
Into the documentation for this function, add a jsdoc line for what `service` is. It will help as well if examples (or the set of allowed values, if able) is included.

e.g., change as necessary
```javascript
/**
 * @param {string} service The service to create a ruleset.
 *    Valid values are "cloud.firestore" and "firebase.storage"
 */
```